alf/experience_replayers/replay_buffer.py

```diff
@@ -57,11 +60,11 @@ def _create_buffer(tensor_spec):
         with tf.device(self._device):
             self._current_size = tf.Variable(
                 name=name + "/size",
-                initial_value=tf.zeros((num_environments, ), tf.int32),
+                initial_value=tf.zeros((num_environments, ), tf.int64),
```

Comment:
Why int64? int32 seems enough

Reply:
I changed it to int64 because tf needs int64 to access large tensors (>2GB or 4GB)